### ISSUE TITLE
create a new task

### ISSUE THREAD

**Issue by balavadze** [IsabelKimTina/IsabelKimTina#14]
React router:

- [x]     create a new page
- [x]     a link in the navbar to the new page

Form
- [ ]  create a component with a form

The form should have:
        

- [ ] Input fields (only placeholders, no pre-filled values!) (no field for id!)
- [ ] One field for an image upload
- [ ] validation
- [ ] one button that triggers the onSubmit function.

an onSubmit function:
In the onSubmit function you should:

- [ ] send the data from the form to the backend fetch with POST. 

- [ ] If everything is working, display a message or alert on the page and a link back to the main page.

Once the backend is done:

- [ ] Without changing the Container, you should see the new item as a card.
